src/rules/order-properties.ts

```diff
@@ -69,11 +69,19 @@ export const rule = createRule<Options>({
 
 				const { properties } = ast.body[0].expression;
 
+				// we only need to run fix once
+				let withFix = true;
```

Comment:
[Question] Is it _bad_ to have a fixer attached to each report, though? It does feel a little confusing to me how only the first one has the prompt for auto-fixing. And perhaps also inconvenient that only one of the potentially many reports will give the editor prompt to fix?